src/UCommerce.SiteCore.Installer/Steps/MoveFileIfTargetExist.cs

```diff
@@ -1,26 +1,23 @@
 ﻿using System.Collections.Specialized;
 using System.IO;
+using System.Threading.Tasks;
 using System.Web.Hosting;
-using Sitecore.Install.Framework;
 using Ucommerce.Installer;
 
 namespace Ucommerce.Sitecore.Installer.Steps
 {
-	public class MoveFileIfTargetExist : IPostStep
+	public class MoveFileIfTargetExist : IStep
 	{
 		private readonly bool _backupTarget;
 		private readonly FileMoverIfTargetExist _command;
 
-		public MoveFileIfTargetExist(string sourceVirtualPath, string targetVirtualPath, bool backupTarget)
+		public MoveFileIfTargetExist(FileInfo sourceVirtualPath, FileInfo targetVirtualPath, bool backupTarget)

```

Comment:
Could the parameter names "sourceVirtualPath" and "targetVirtualPath" be changed to something like "sourceFile" or "targetFile", since FileInfo now references the file and not the path